login-script.php

```diff
@@ -4,33 +4,42 @@
     include("./classes/functions.php");
 
     $email = sanitize($_POST["email"]);
-    // $passwd = sanitize($_POST["password"]);
+    $passwd = sanitize($_POST["passwd"]);
+    // $pass = $sql = "SELECT * from `password` WHERE `passwd` = '$passwd'";
 
     $sql = "SELECT * from `password` WHERE `email` = '$email'";
+    // $passwdDB = "SELECT * from `password` WHERE `passwd` = '$passwd'";
+    
+    
     $result = mysqli_query($conn, $sql);
-
+    $record = mysqli_fetch_assoc($result);
+    $encpass = $record["passwd"];
```

Comment:
With the registration process, there is also a salt created. A salt is a random 10 letter string, also saved in our database. You need this to verify the password as well.